cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -183,7 +181,13 @@ func gitIndex(c gitIndexConfig, o *indexArgs, sourcegraph Sourcegraph, l sglog.L
 
 	buildOptions := o.BuildOptions()
 
-	ctx, cancel := context.WithTimeout(context.Background(), indexTimeout)
+	// indexingTimeout defines how long the indexserver waits before killing an indexing job.
+	indexingTimeout := getEnvWithDefaultDuration("INDEXING_TIMEOUT", defaultIndexingTimeout)
```

Comment:
Should we make this part of indexArgs and set it in `Server.indexArgs`?

Reply:
I took a closer look at this, and decided to go with your suggestion, to match what we do for other debugging options like `SKIP_SYMBOLS_REPOS_ALLOWLIST`. I think it's best to be consistent, even though that means you need to restart the server to pick up new config.